src/go/rpk/pkg/serde/embed/embed_test.go

```diff
@@ -0,0 +1,44 @@
+package embed
+
+import (
+	"io/fs"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestEmbeddedFiles(t *testing.T) {
+	t.Run("Test Embedded files in rpk, equal to Redpanda", func(t *testing.T) {
+		// /src/v/pandaproxy/schema_registry/protobuf
+		redpandaProtoFS := os.DirFS("../../../../../v/pandaproxy/schema_registry/protobuf/")
+		redpandaMap := make(map[string]string)
+		err := fs.WalkDir(redpandaProtoFS, ".", func(path string, d fs.DirEntry, err error) error {
+			if err != nil {
+				return err
+			}
+			if d.IsDir() {
+				return nil
+			}
```

Comment:
Same comment here I guess :D

Reply:
Fixed in https://github.com/redpanda-data/redpanda/compare/0404d15560b25c7026b593382887829c77a03ceb..044debc20df7bbda0e0bf68653119370a0f08b4a 

Thanks.